storage/innobase/xtrabackup/test/run.sh

```diff
@@ -372,6 +372,15 @@ EOF
 {
  "read_local_config": true
 }
+EOF
+  fi
+
+  FILE="$MYSQL_BASEDIR/lib/plugin/component_keyring_kms.cnf"
+  if [[ ! -f "${FILE}" ]]; then
+    cat <<EOF > "${FILE}"
+{
+  "read_local_config": true
```

Comment:
why does this config value mean? should we test with false too?

Reply:
This means config values will be read from instance datadir instead of binary / lib dir. 
Since all tests share the same xtrabackup / mysqld binary, we must use the local flag, otherwise, all tests will be reading/writing and deleting the same keyring files.

Reply:
Thanks